### PR TITLE
Integration 

### PR DESCRIPTION
**Pull request by Rythmag**
1. Call findAndStoreError function after storing a file
2. Add a delay before calling findAndStoreError
3. Use encoded name for stackTraceservlet call
4. Change in text input css to avoid text wrap
5. Add NO_ERROR_FOUND_MSG as document to prevent index not found exception for error file

### CODE REVIEW

File: src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java [hunk 1]
```diff
@@ -41,6 +41,7 @@ public String checkAndStoreFileAndUrlLog(HttpServletRequest request,
             indexName = storeLogs.getUniqueIndexName(indexName);
             final String response = storeFileAndUrlLogs(
                 request, indexName, fileContent, isUrl);
+            storeLogs.findErrorsInIndex(indexName);
```
Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java:44]: Have we tested that the errorIndex is formed?

Comment (nishantnahata) [inline src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java:44]: I guess storeFileAndUrlLogs internally calls storeLogs.checkAndStoreLog with offset. This is not the right place to call it.
Call it in checkAndStoreLog with offset

File: src/main/java/com/google/error_analyzer/backend/FilterErrors.java [hunk 2]
```diff
@@ -42,6 +45,10 @@
                 documentList.add(document);
             }
         }
+        if (documentList.build().size() == 0) {
+            Document document = new Document("1", 1, NO_ERROR_FOUND_MSG);
```
Comment (nishantnahata) [inline src/main/java/com/google/error_analyzer/backend/FilterErrors.java:49]: is it necessary?

File: src/main/java/com/google/error_analyzer/backend/StoreLogs.java [hunk 3]
```diff
@@ -108,4 +110,13 @@ public String getUniqueIndexName(String indexName) throws IOException {
         return nextIndexName;
     }
 
+    public void findErrorsInIndex(String indexName) {
+        try {
+            TimeUnit.SECONDS.sleep(1);
```
Comment (nishantnahata) [inline src/main/java/com/google/error_analyzer/backend/StoreLogs.java:115]: why sleep?

File: src/main/java/com/google/error_analyzer/backend/StoreLogs.java [hunk 4]
```diff
@@ -52,6 +53,7 @@ public String checkAndStoreLog(HttpServletRequest request, String fileName,
             indexName = getUniqueIndexName(indexName);
             final String response = storeLog(
                 request, indexName, log, OFFSET_FOR_PLAIN_TEXT );
+            findErrorsInIndex(indexName);
```
Comment (nishantnahata) [inline src/main/java/com/google/error_analyzer/backend/StoreLogs.java:56]: Update the test for checkAndStoreLog

Comment (shudhatmajain) [inline src/main/java/com/google/error_analyzer/backend/StoreLogs.java:56]: Have we tested (UT) that the error index is created?

File: src/main/java/com/google/error_analyzer/backend/FilterErrors.java (hunk 2 above)
Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/FilterErrors.java:49]: Empty bulk request gives an exception and error index is not created

File: src/main/java/com/google/error_analyzer/backend/StoreLogs.java (hunk 3 above)
Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/StoreLogs.java:115]: If the findErrorsInIndex is run right after storing the result is always 0 errors

File: src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java (hunk 1 above)
Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java:44]: That would call the same function every time after storing 10k lines

Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java:44]: That would call the finAndStoreErrors function multiple times

File: src/main/java/com/google/error_analyzer/backend/StoreLogs.java (hunk 4 above)
Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/StoreLogs.java]: test added

File: src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java (hunk 1 above)
Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/FileAndUrlLogs.java]: test added

File: src/main/java/com/google/error_analyzer/backend/StoreLogs.java (hunk 4 above)
Comment (Rythmag) [reply src/main/java/com/google/error_analyzer/backend/StoreLogs.java]: test added

**Pull request merged**